Tests/Foundation/Tests/TestAttributedString.swift

```diff
@@ -2144,7 +2144,7 @@ E {
         
         let expected =
             AttributedString("This is a ") +
-            AttributedString("mispelled", attributes: AttributeContainer().misspelled(true)) +
+            AttributedString("misspelled", attributes: AttributeContainer().misspelled(true)) +
```

Comment:
Seems like this is the intended behavior for me.

To test "pel" to "spell" and "mispelled" to "misspelled" both got corrected.

But since FixableAttribute is not implemented now and this test is not enabled, only the author know which is the intended behavior.